src/routes/v1/friendships/history.rs

```diff
@@ -0,0 +1,101 @@
+use actix_http::HttpMessage;
+use actix_web::{
+    get,
+    web::{self, Data},
+    HttpRequest, HttpResponse,
+};
+use chrono::NaiveDateTime;
+use serde::{Deserialize, Serialize};
+use uuid::Uuid;
+
+use crate::{
+    components::app::AppComponents,
+    entities::friendship_history::FriendshipHistory,
+    middlewares::check_auth::UserId,
+    routes::v1::{
+        error::CommonError,
+        friendships::{errors::FriendshipsError, types::MessageRequestEventResponse},
+    },
+};
+
+use super::types::MessageRequestEvent;
+
+#[derive(Deserialize, Serialize)]
+pub struct RequestEventRequestBody {
+    pub timestamp_from: i64, // timestamp in milis
+    pub timestamp_to: i64,   // timestamp in milis
+}
+
+#[get("/v1/friendships/{friendshipId}/request-events/messages")]
```

Comment:
Ideally, I would like to narrow the query of the history of `request` events between two timestamps: `timestamp_from` and `timestamp_to`. I thought it would be a good idea to define the `RequestEventRequestBody` struct with the two timestamps in milliseconds, since that is the information that the kernel will be able to provide us.